lib/registry.js

```diff
@@ -16,7 +17,9 @@ export function define(id, spec) {
   if (get(id)) {
     console.warn(`Overwriting spec ${getName(id)}`)
   }
-  spec.name = getName(id)
+  if (!spec.name && typeof spec !== "symbol") {
```

Comment:
Why is it bad to use global Symbols? I think it's more in line with what people expect from string constants as keys to data structures. Because "matt" is equal to "matt" in any module, namespace, closure, etc. you can thing of strings being globally defined symbols.

I also agree that Symbol should be the only key type but that's a harder argument to make as it will require a shared polyfill if one isn't using babel or ES6.